CHANGELOG.rst

```diff
@@ -16,6 +16,12 @@ In development
   is now exposed to let st2 know about the location of web ui. This is used to construct the
   web url. Chatops messages can now display the history url from the execution model they
   get as response to API calls.
+* Update action runner to use two internal green thread pools - one for regular (non-workflow) and
+  one for workflow actions. Sizes of both of those pools are also user configurable inside the
```

Comment:
How about we re-write these sentences like this:
* Both pool sizes are user-configurable. This should help increase the throughput of a single action runner when the system is not over-utilized. It can also help prevent deadlocks which may occur when using delay policies with action-chain workflows. (improvement)